### ISSUE TITLE
Leaving craft function activates after leaving craft.

### ISSUE THREAD

**Issue by danielcbailey** [msummers123/Movecraft-3#187]
I have been testing out ships in my mock ship factory on build to see if they fit, and when sending them to cruise off into the distance, I leave the craft, and I get the "You have left your craft" message a few times, odd. But when I sit on a surface of my factory, I get the message again. When I just off a surface of my factory I also get the message, however when I stand still I do not get a repeated message. These blocks are far from the crafts bounds, and I had to stop the ship with /release as it would not release after 30 seconds.
![2016-04-06_14 23 01](https://cloud.githubusercontent.com/assets/9934819/14328227/c9546342-fc03-11e5-9e9e-ed9518b7c4b7.png)

**Comment by pistols12** [comment]
You have the most recent movecraft version correct? I would start by verifying that it's doing the same thing on all of your craft files. If so then maybe I would check to see the time that is set for I'm piloting. Or just reinstall the newest version again.

**Comment by pistols12** [comment]
Also, if you have a craft file that you want me to test I could do that for you. My email is LoganpClayton@gmail.com also, Dan from ap?

**Comment by TheRealBotcrusher** [comment]
I have reproduced this bug before, sometimes when cruising it resets the
counter every cruise. It seems to mostly happen when you are moving quickly
and manage to glitch off the ship.

**Comment by danielcbailey** [comment]
Yes, I am danielcbailey >.> Hence the destrian ship. It is BaccaYarro's Public Build Server and thus uses the newest movecraft. And no, I walked off the ship, nothing special.

**Comment by pistols12** [comment]
Well I've definitely experienced this with my servers Ram craft. It seems to have been post 1.9 that I experienced it. So if it is a random intermittent issue, I would suggest looking at what 1.9 that could possibly be a culprit. Have you guys upgraded to 1.9 yet? Or are you still in the 1.8 dust.

**Comment by danielcbailey** [comment]
Dust.

**Comment by pistols12** [comment]
Okay, interesting. I got this error again today will jumping off my plow to let it continue to harvest my crops while I did other things. It was spamming this error and I doubt that it was going to release because one of my Harvest blades was chopped off, so that the wheat was stopping the craft. For me it's possible that I left right when it made collision with something. And perhaps it got stuck in a logic Loop for you it doesn't look like that was possible. Good to know that it stretches across both versions though. This is a new problem to me, so I'm wondering if something got changed in a newer version of movecraft that could have to do with this. We've ruled out the versions of our servers causing the issue, the next thing I can think of is the plug-in itself. Specially since you had an aircraft and doing this and I had a ground plow

**Comment by pistols12** [comment]
I'm still not strong in the coding world, but it seems like there could be a chance movecraft gets the craft stuck in a logical Loop instead of releasing it. Or instead of realizing that there's no person on it to be able to release it

**Comment by pistols12** [comment]
Our servers are very different so it's hard to imagine that anyone plugin could be causing the issue as well. Other than movecraft itself

**Comment by pistols12** [comment]
Any one plugin*
